Dockerfile

```diff
@@ -40,7 +40,12 @@ EXPOSE $PUMA_PORT
 RUN addgroup -g $APPUID -S appgroup && \
     adduser -u $APPUID -S appuser -G appgroup -h /app
 
-RUN apk add --update --no-cache tzdata postgresql-dev
+RUN apk add \
+      --update \
+      --no-cache \
+      tzdata \
+      postgresql-dev \
+      postgresql-client
```

Comment:
As a matter of principle I like to keep the image footprint to a minimum needed for the runtime environment.
But I also appreciate the convenience this brings.

On that trade-off I think it's OK to have this for now. We can always remove it at a later time when this need is no longer relevant.